exercises/yacht/yacht_test.py

```diff
@@ -4,6 +4,7 @@
 
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.2.0
 
+
```

Comment:
I tried to format the test file manually but this caused Travis to fail because of the `generate_tests ` check. I found [no better way to prettify output](https://stackoverflow.com/questions/35775207/remove-unnecessary-whitespace-from-jinja-rendered-template) so I'm leaving it like that.

Reply:
This spacing is introduced by `black`, which is used to autoformat generated code.